dag.yml

```diff
@@ -159,6 +159,13 @@ steps:
   data://grapher/un/2022-07-11/un_wpp:
     - data://garden/un/2022-07-11/un_wpp
 
+  # Internet
+  data://garden/owid/latest/internet:
```

Comment:
Why should this dataset be under "owid"?

Reply:
So far I got the impression that namespaces always were entities/orgs. So in this case we are the ones creating the dataset...

What do you propose? Having a namespace like 'technology'?